App/src/components/NewCalendar.js

```diff
@@ -97,13 +97,22 @@ function NewCalendar({
   users, // the list of users to show
   selectedUsers = [], // the list of users to highlight
   selectedDate = null // the date to highlight those users
+  // shiftsClicked
 }) {
   const dates = Array.from({ length: 7 }, (_, index) => {
     const date = new Date(startDate);
     date.setDate(startDate.getDate() + index);
     return date;
   });
 
+  const buttonCss = {
+    background: "none",
+    border: "none",
+    cursor: "pointer",
+    display: "flex",
+    alignItems: "center"
```

Comment:
Also add ` width: "100%"` here to ensure the button uses the whole size of the table cell.
```suggestion
    alignItems: "center",
    width: "100%"
```